common/src/main/java/com/gregtechceu/gtceu/data/recipe/misc/MachineRecipeLoader.java

```diff
@@ -353,8 +353,8 @@ private static void registerAlloyRecipes(Consumer<FinishedRecipe> provider) {
                     .save(provider);
         }
 
-        COMPRESSOR_RECIPES.recipeBuilder("carbon_mesh").inputItems(CARBON_FIBERS, 2).outputItems(CARBON_MESH).duration(100).save(provider);
-        COMPRESSOR_RECIPES.recipeBuilder("carbon_fiber_plate").inputItems(CARBON_MESH).outputItems(CARBON_FIBER_PLATE).save(provider);
+        COMPRESSOR_RECIPES.recipeBuilder("carbon_mesh").inputItems(CARBON_FIBERS, 2).outputItems(CARBON_MESH).duration(100).EUt(2).save(provider);
+        COMPRESSOR_RECIPES.recipeBuilder("carbon_fiber_plate").inputItems(CARBON_MESH).outputItems(CARBON_FIBER_PLATE).duration(400).EUt(2).save(provider);
```

Comment:
Default duration here should be 200, unless this was a manual duration lost in the port

Reply:
ok i'll remove the durations